explorer/pkg/gex/gex.go

```diff
@@ -0,0 +1,75 @@
+package gex
+
+import (
+	"archive/tar"
+	"bytes"
+	"compress/gzip"
+	"context"
+	"io"
+	"os"
+
+	"github.com/ignite/cli/ignite/pkg/cmdrunner/exec"
+	"github.com/ignite/cli/ignite/pkg/cmdrunner/step"
+	"github.com/ignite/cli/ignite/pkg/localfs"
+	"github.com/ignite/ignite-files/gex"
```

Comment:
Is this because it cannot be used as a library?
Say it was usable as a library, would the design still need to rely on using a pre-built binary?